department-of-veterans-affairs/va.gov-cms
# VACMS-16853: next-build git information for content release

## Description

closes #16853 

## Testing done

Manually on my local machine  and also Tugboat using the QA steps provided below.

## Screenshots

<img width="778" alt="Screenshot 2024-01-22 at 4 31 00 PM" src="https://github.com/department-of-veterans-affairs/va.gov-cms/assets/3640707/96109a8c-a23a-4b61-9085-4d294ce4b0e3">

## QA steps

1. Make sure the next-build repo is cloned into the right place. Should be done with `composer va:next:install`
2. Make sure the `GITHUB_TOKEN` environmental variable is set with a valid token.
3. Log into the CMS and go to `/admin/content/deploy/next_git`
   - [ ] Confirm `next-build` is in the first form radio instead of `content-build`
4. Select the "Select a different next-build branch/pull request - for example, to see your content in a newer frontend design." radio on the first radio group.
5. Look for some PRs on https://github.com/department-of-veterans-affairs/next-build/pulls
6. Type the first couple words of each PR into the autocomplete box
   - [ ] Confirm PR 1 shows up and can be selected
   - [ ] Confirm PR 2 shows up and can be selected

### Definition of Done

- [x] Tests have been updated if necessary.

### Select Team for PR review
